README.md

```diff
@@ -217,6 +217,21 @@ end
 Gollum::Hook.unregister(:post_commit, :hook_id)
 ```
 
+Register or unregister a hook to be called after the wiki is initialized:
+
+```ruby
+Gollum::Hook.register(:post_wiki_initialize, :hook_id) do |wiki|
+  # Your code here
+end
+
+Gollum::Hook.unregister(:post_wiki_initialize, :hook_id)
+```
+
+A combination of both hooks can be used to pull from a remote after
+`:post_wiki_initialize` and push from a remote after `:post_commit` which in
```

Comment:
should this be `push to a remote` (not sure)?

Reply:
Oh, you're right, it's a typo. Fixed, amended as cd40e25b39a9df54ab4063f4346383c81fc03f30 and force pushed. Thanks for heads-up!

Reply:
In fact, it's kind of nonsensical to suggest pull and push, then add a heads-up to make remote read-only – in this case, push is sufficient. I've therefore rephrased things:

> A combination of both hooks can be used to pull from a remote after
`:post_wiki_initialize` and push to a remote after `:post_commit` which in
effect keeps the remote in sync both ways. Keep in mind that it may not be
possible to resolve all conflicts automatically.